plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -459,6 +461,292 @@ public void testArrayFields()
                 "VALUES ('hello', 20, 30, 60, NULL)");
     }
 
+    @Test
+    public void testAsRawJson()
+            throws IOException
+    {
+        String indexName = "raw_json";
```

Comment:
Use `TestTable.randomTableSuffix()` to avoid any collisions with table names in future.